### PR TITLE
reference/tools: refine mydumper doc

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Update an FAQ: `时间类型的数据` -> `TIMESTAMP 类型的数据`
- Fix incorrect punctuation
- Improve the format and wording of mydumper doc

### What is the related PR or file link(s)? 
N/A

### Which version does your change affect? 
dev, v3.1, v3.0, v2.1

### CODE REVIEW

**Comment by juliezhang1112** [comment]
PTAL @WangXiangUSTC Thanks~

File: dev/reference/tools/mydumper.md [hunk 1]
```diff
@@ -79,109 +79,111 @@ Mydumper 首先计算 `min(_tidb_rowid)` 和 `max(_tidb_rowid)`，然后按照 `
...
+执行如下命令：
 
 {{< copyable "shell-regular" >}}
 
 ```bash
 ./bin/mydumper -V
 ```
 
-输出如下：
+输出结果中包含 `githash`（示例输出中的 `d3e6fec8b069daee772d0dbaa47579f67a5947e7`）的即为 PingCAP 优化的版本：
```
Comment (yikeke) [inline dev/reference/tools/mydumper.md]:
```suggestion
如果输出结果中包含 `githash`（如下列示例输出中的 `d3e6fec8b069daee772d0dbaa47579f67a5947e7`），则使用的 Mydumper 为 PingCAP 优化的版本。
```

File: dev/reference/tools/mydumper.md [hunk 2]
```diff
@@ -79,109 +79,111 @@ Mydumper 首先计算 `min(_tidb_rowid)` 和 `max(_tidb_rowid)`，然后按照 `
...
+### 如何配置 Mydumper 的参数 `-F, --chunk-filesize`？
 
-Mydumper 在备份时会根据这个参数的值把每个表的数据划分成多个 `chunk`，每个 `chunk` 保存到一个文件中，大小约为 `chunk-filesize`。根据这个参数把数据切分到多个文件中，这样就可以利用 Loader/TiDB-Lightning 的并行处理逻辑提高导入速度。 如果后续使用 Loader 对备份文件进行恢复，建议把该参数的值设置为 64（单位 MB）；如果使用 TiDB-Lightning 恢复，则建议设置为 256（单位 MB）。
+Mydumper 在备份时会根据这个参数的值把每个表的数据划分成多个 `chunk`，每个 `chunk` 保存到一个文件中，大小约为 `chunk-filesize`。根据这个参数把数据切分到多个文件中，这样就可以利用 Loader/TiDB-Lightning 的并行处理逻辑提高导入速度。如果后续使用 Loader 对备份文件进行恢复，建议把该参数的值设置为 `64`（单位 MB）；如果使用 TiDB-Lightning 恢复，则建议设置为 `256`（单位 MB）。
 
-### 如何配置 Mydumper 的参数 `-s --statement-size`?
+### 如何配置 Mydumper 的参数 `-s --statement-size`？
 
-Mydumper 使用该参数控制 `Insert Statement` 的大小，默认值为 1M 左右。使用该参数来尽量避免在恢复数据时报以下错误：
+Mydumper 使用该参数控制 `Insert Statement` 的大小，默认值为 `10000000`(约 1 MB）。使用该参数来尽量避免在恢复数据时报以下错误：
```
Comment (yikeke) [inline dev/reference/tools/mydumper.md]:
```suggestion
Mydumper 使用该参数控制 `Insert Statement` 的大小，默认值为 `10000000`（约 1 MB）。使用该参数来尽量避免在恢复数据时报以下错误：
```

File: dev/reference/tools/mydumper.md [hunk 3]
```diff
@@ -79,109 +79,111 @@ Mydumper 首先计算 `min(_tidb_rowid)` 和 `max(_tidb_rowid)`，然后按照 `
...
+### 如何配置 Mydumper 的参数 `-F, --chunk-filesize`？
 
-Mydumper 在备份时会根据这个参数的值把每个表的数据划分成多个 `chunk`，每个 `chunk` 保存到一个文件中，大小约为 `chunk-filesize`。根据这个参数把数据切分到多个文件中，这样就可以利用 Loader/TiDB-Lightning 的并行处理逻辑提高导入速度。 如果后续使用 Loader 对备份文件进行恢复，建议把该参数的值设置为 64（单位 MB）；如果使用 TiDB-Lightning 恢复，则建议设置为 256（单位 MB）。
+Mydumper 在备份时会根据这个参数的值把每个表的数据划分成多个 `chunk`，每个 `chunk` 保存到一个文件中，大小约为 `chunk-filesize`。根据这个参数把数据切分到多个文件中，这样就可以利用 Loader/TiDB-Lightning 的并行处理逻辑提高导入速度。如果后续使用 Loader 对备份文件进行恢复，建议把该参数的值设置为 `64`（单位 MB）；如果使用 TiDB-Lightning 恢复，则建议设置为 `256`（单位 MB）。
 
-### 如何配置 Mydumper 的参数 `-s --statement-size`?
+### 如何配置 Mydumper 的参数 `-s --statement-size`？
 
-Mydumper 使用该参数控制 `Insert Statement` 的大小，默认值为 1M 左右。使用该参数来尽量避免在恢复数据时报以下错误：
+Mydumper 使用该参数控制 `Insert Statement` 的大小，默认值为 `10000000` (约 1 MB）。使用该参数来尽量避免在恢复数据时报以下错误：
```
Comment (yikeke) [inline dev/reference/tools/mydumper.md]:
```suggestion
Mydumper 使用该参数控制 `Insert Statement` 的大小，默认值为 `10000000`（约 1 MB）。使用该参数来尽量避免在恢复数据时报以下错误：
```
Use Chinese parentheses.

**Comment by juliezhang1112** [comment]
PTAL again @yikeke Thanks~

**Pull request merged**